From aaaabf1ea84d9c5ae9cf50a306b79f15f44f5b35 Mon Sep 17 00:00:00 2001
From: Carson Black <uhhadd@gmail.com>
Date: Sat, 19 Sep 2020 12:21:33 -0400
Subject: [PATCH] Revert "Fix: avatar loading"

This reverts commit 338d91be7254099c1ad4c1f475acb6368df2b114, which
introduced a visual regression in the users KCM among other places
that use the Avatar by causing the image to fail to resize according
to the size of the avatar component.
---
 src/controls/Avatar.qml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/controls/Avatar.qml b/src/controls/Avatar.qml
index a4f59448..775e5987 100644
--- a/src/controls/Avatar.qml
+++ b/src/controls/Avatar.qml
@@ -133,9 +133,6 @@ QQC2.Control {
             mipmap: true
             smooth: true
 
-            sourceSize.width: avatarRoot.implicitWidth
-            sourceSize.height: avatarRoot.implicitHeight
-
             fillMode: Image.PreserveAspectFit
             anchors.fill: parent
         }
-- 
GitLab

